quartz/components/scripts/checkbox.inline.ts

```diff
@@ -0,0 +1,28 @@
+document.addEventListener("nav", () => {
+  document.querySelectorAll(".task-list-item").forEach(addToggleHandler)
+})
+
+function addToggleHandler(listItem: Element) {
+  const toggleCheckbox = (e: any) => {
+    const updatedCheckboxState: boolean = e.target.checked
+    const checkboxId = e.target.id
+    localStorage.setItem(`isCheckbox-${checkboxId}-enabled`, JSON.stringify(updatedCheckboxState))
```

Comment:
```suggestion
    localStorage.setItem(`is-checkbox-${checkboxId}-enabled`, JSON.stringify(updatedCheckboxState))
```

prefer kebab for storage items